app/routes/models/calculation.js

```diff
@@ -128,13 +142,6 @@ function fillGaps (results, data, formatType) {
   return data
```

Comment:
Lines 128 and 129 can be converted to the spread operator instead of using deprecated .apply method:

  const maxSchemeYear = Math.max(Math, ...schemeYears)
  const minSchemeYear = Math.min(Math, ...schemeYears)